README.md

```diff
@@ -264,5 +264,29 @@ For example:
   RtspTcpPort = "8554"
 ```
 
+
+### CameraStatus REST API Endpoint
+Use the following query to determine the status of the camera.
+
+URL parameter:
+- **DeviceName**: The name of the camera
```

Comment:
The parameter is not just `device name` but also `InputVideoIndex`.

Reply:
`InputIndex` is the query parameter :-)